src/Lean/Parser/Term.lean

```diff
@@ -51,6 +51,7 @@ namespace Term
 /- Built-in parsers -/
 
 @[builtinTermParser] def byTactic := leading_parser:leadPrec ppAllowUngrouped >> "by " >> Tactic.tacticSeq
+def byTactic' := leading_parser "by " >> Tactic.tacticSeq
```

Comment:
Could you please add a comment explaining why `byTactic'` has been added? It can be based on the PR comment.